cgroup/cgroup.go

```diff
@@ -82,7 +89,26 @@ func (m manager) write(path, data string) error {
 
 // generatePath generates a path within the cgroup like /<mount>/<kind>/<path (kubepods)>
 func (m manager) generatePath(kind string) string {
-	return fmt.Sprintf("%s%s/%s", m.mount, kind, m.path)
+	generatedPath := fmt.Sprintf("%s%s/%s", m.mount, kind, m.path)
+	foundPath := fmt.Sprintf("%s%s/%s", m.mount, kind, m.cgroup.Paths[kind])
+
+	if generatedPath != foundPath {
+		m.log.Infow("containerd generated path does not match found cgroup path",
+			"generatedPath", generatedPath,
+			"foundPath", foundPath,
+		)
+	}
+
+	if _, err := os.Stat(generatedPath); os.IsNotExist(err) {
+		m.log.Infow("containerd generated path does not exist, using found path instead",
+			"generatedPath", generatedPath,
```

Comment:
Do we want this line?

Reply:
I think so! I am pretty sure I want to know when these paths aren't the same, whether or not the generated path exists